packages/orbit-components/src/NavigationBar/README.md

```diff
@@ -35,9 +35,4 @@ Table below contains all types of the props available in the NavigationBar compo
 | hideOnScroll       | `boolean`               | `true`                   | Turn on or off hiding navigation bar on scroll                                                              |
 | openTitle          | `string`                | `"Open navigation menu"` | Property for passing translation string to open Button.                                                     |
 | bottomStyle        | `"shadow" \| "border"`  | `"shadow"`               | Property for setting bottom style of NavigationBar.                                                         |
-| ariaLabel          | `string`                | `"navigation"`           | Optional prop for `aria-label` value (accessibility).                                                       |
```

Comment:
I was thinking more about this and I guess we don't need to remove `ariaLabel` to avoid BC.

But at the same time, I really don't think it's necessary here and keeping unnecessary props might be confusing and we would need to remove it eventually anyway 🤔

Reply:
hmm, I think the situation is similar to my [PR](https://github.com/kiwicom/orbit/pull/4682) where I did vice versa, I removed unnecessary prop :D 

in my view, it's better to remove it and have clear code, although it will be BC.